Jenkinsfile

```diff
@@ -259,6 +267,14 @@ def evaluateJson(String json, String gpath){
     return new groovy.text.GStringTemplateEngine().createTemplate(gpath).make(json:ojson).toString()
 }
 
+@NonCPS
+def evaluateJsonList(String json){
```

Comment:
Do you need the separate function now? Looks simple enough it could be inline now?